package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "disney-villains-api",
+  "version": "1.0.0",
+  "description": "In this two part exrcise you are tasked with create a REST API for accessing Disney Villain data, complete with test suite.",
+  "main": ".eslintrc.js",
+  "scripts": {
+    "dev": "./node_modules/.bin/nodemon index.js",
+    "lint": "./node_module/.bin/eslint -- format codeframe",
+    "lint:fix": "./node_module/.bin/eslint -fix -- format codeframe ",
+    "test": "./node_modules/.bin/mocha tests/**/**/.test.js"
```

Comment:
This command doesn't work, should be `"test": "./node_modules/.bin/mocha test/**/*.test.js"`